controllers/precache.go

```diff
@@ -334,24 +309,23 @@ func (r *ClusterGroupUpgradeReconciler) getPrecachingSpec(
 					return *new(ranv1alpha1.PrecachingSpec), nil
 				}
 				spec.PlatformImage = fmt.Sprintf("%s", image)
-				r.Log.Info("[getPrecachingSpec]", "ClusterVersion image", image)
+				r.Log.Info("[extractPrecachingSpecFromPolicies]", "ClusterVersion image", image)
 			case "Subscription":
```

Comment:
It would be nice to have these cases as constants in utils: "ClusterVersion", "Subscription",...

Reply:
Done